doc/man3/EVP_PKEY_decrypt.pod

```diff
@@ -49,15 +49,16 @@ Decrypt data using OAEP (for RSA keys):
  #include <openssl/rsa.h>
 
  EVP_PKEY_CTX *ctx;
+ ENGINE *eng;
  unsigned char *out, *in;
  size_t outlen, inlen;
  EVP_PKEY *key;
 
  /*
-  * NB: assumes key in, inlen are already set up
+  * NB: assumes eng, key in, inlen are already set up
```

Comment:
I'd like to argue in favour of listing variables in order of their appearance in snippet. Also there is comma missing after `key`, and I wonder if you could add that too, while we are at it. Also, I'm not native English speaker, just wonder if `assumes` is right in the context. I mean I wouldn't use `assumes`, but rather `assume` or `it's assumed`. No?

Reply:
Sure. I will add that comma and use the `key, eng, in, inlen` order (if I understand how you mean to order them).
I am not native either, but I don't see anything wrong in `assumes` in this context, though it would be more clear if the subject was in the sentence explicitly mentioned (`the function assumes ...`).

Reply:
I am a native speaker and "assumes" sounds fine to me.

Reply:
Yes, this is short for saying "the implementation of this example assumes that the following parameters have been set by the calling code"